Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -294,7 +294,7 @@ public enum FeatureFlag: String, CaseIterable {
         case .encourageAccountCreation:
             true
         case .notificationsRevamp:
-            false
+            true
```

Comment:
This will be reverted before merge, is just to make it easier to test on first install the skipping of notifications permissions.